frontend/src/components/WofSmallError.vue

```diff
@@ -0,0 +1,72 @@
+<template>
+<div class="small-error">
+    <div class="small-error__contener">
+        <div class="small-error__icon">
+            <wof-icon @mouseover="hover = true" @mouseleave="hover = false" icon="wof-warning-circle-fill" :size="this.size"/>
+        </div>
+        <div class="small-error__wrap-text">
+            <div class="small-error__content" v-if="hover">
+                <slot></slot>
+            </div>
+        </div>
+    </div>
+</div>
+</template>
+
+<script>
+export default {
+  name: "WofSmallError",
+
+  props: {
+      size: Number
+  },
+  data() {
+    return {
+      hover: false
```

Comment:
I think that you could do display on hover with CSS only. Search the internet for solution.